components/org.wso2.identity.outbound.adapter.websubhub/src/test/java/org/wso2/identity/outbound/adapter/websubhub/util/TestUtils.java

```diff
@@ -0,0 +1,111 @@
+package org.wso2.identity.outbound.adapter.websubhub.util;
```

Comment:
Let's add the license header.

Reply:
Resolved via [4dc618f](https://github.com/wso2-extensions/identity-outbound-adapters/pull/28/commits/4dc618fd30ccdd17e9f185a3f098bc25f7f1c344)